src/pages/home/Home.jsx

```diff
@@ -39,7 +39,7 @@ function Home() {
         <h1 className={styles.anton}>and Connect People</h1>
       </section>
       <section className={styles.button}>
-          <button className={styles.signupButton} onClick={handleSignupClick}><p className={styles.center}>Sign up for help</p ></button>
+          <button className={styles.signupButton} onClick={handleSignupClick}><p className={styles.center}>Join eco - friends</p ></button>
```

Comment:
can you change that to `eco-friends` instead?

Reply:
I will capitalized the E for eco-friends.

Reply:
I guess that is fine with me

Reply:
DONE